src/__experimental__/components/radio-button/radio-button-group.spec.js

```diff
@@ -79,7 +77,7 @@ describe('RadioButtonGroup', () => {
           const radioGroup = shallow(
             <RadioButtonGroup
               groupName={ groupName }
-              label='Test RadioButtonGroup Label'
+              legend='Test RadioButtonGroup Label'
```

Comment:
```suggestion
              legend='Test RadioButtonGroup Legend'
```

Reply:
resolved in [https://github.com/Sage/carbon/pull/2395/commits/8ad77ae94edd822af3a2c8bee9159daa445c16b8](8ad77ae)